app/views/bills/_form.html.erb

```diff
@@ -22,11 +34,13 @@
     <%= f.text_field :usage, :id => "usage" %>
   </div>
 
-  <%= f.fields_for :units do |units_form| %>
-    <div class="field">
-      <%= units_form.label :number_occupants, "Number of Occupants in Your Home or Apartment" %><br>
-      <%= units_form.text_field :number_occupants, :id => "occupants" %>
-    </div>
+  <% if @show_occupants %>
```

Comment:
No I totally agree with not having the instance variable being declared twice...that slipped by me last night and I realized it today. I will move the conditional into the view and see if we can use current_user.unit.present?